news/2022-06-30-community-contributors-2021.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: post
+title: "Community Contributors: 2021"
+date: 2022-06-30 11:00:00 +0000
+---
+
+The osu! community continues on due to the tireless efforts of a mere handful of remarkable individuals. Today, we celebrate their contributions with our highest honour: the badge of a Community Contributor.
+
+![](/wiki/shared/news/2022-06-30-community-contributors-2021/title.jpg)
+
+Clinging on to the traditions of the internet of yore, osu! remains one of the dwindling few places that is truly shaped by its community. From the first time our pink cookie flickered to life in 2007 to the sprawling metropolis of creative minds and dedicated players it has become in 2022, 15 years of osu!'s history has shared one remarkable constant: the community.
+
+Some people take it upon themselves to go simply above and beyond all expectations, and it is upon their labours that the foundation of osu! has been laid all this time.
+
+![Community Contributor badge](https://assets.ppy.sh/profile-badges/contributor.jpg)
+
+We are privileged to recognise more of these incredible people (in no particular order):
+
+## Contributors
+
+[![](/wiki/shared/news/2022-06-30-community-contributors-2021/kasumi-sama.jpg)](https://osu.ppy.sh/users/6177263)
+
+Single-handedly leading the charge forward to elevate the osu!taiko community tournament scene, Kasumi-sama has created a towering legacy under the Global Taiko Showdown (GTS) series. Renowned for their excellent production quality, competitive spirit and considerable reach into entry-level tournaments as a whole, GTS has rapidly established itself as one of the most robust and engaged community projects seen in recent years.
+
+Paired with a keen artistic drive and the wherewithal to engage with many of our Featured Artists and mappers alike to produce new and original content for the community, it is safe to say that Kasumi-sama represents the latest generation of a new era of osu! creators wading knee deep into osu!'s most prominent ethic: making things more awesome.
+
+[![](/wiki/shared/news/2022-06-30-community-contributors-2021/frenzibyte.jpg)](https://osu.ppy.sh/users/14210502)
+
+A regular contributor to ongoing development efforts in osu!(lazer), frenzibyte has made himself well-known in the scene within recent years. His body of work is vast and difficult to accurately explain in a mere few paragraphs, best described instead as making efforts like the iOS version of the game possible once again for thousands of people to enjoy.
```

Comment:
I'm not familiar enough to add anything here myself but this feels a little short. if lazer on iOS is his main thing, it should be written to sound like more of a significant undertaking, and if not, I think you should include more examples of the work he does beyond that

Reply:
the development blurbs are tricky to write because an enormous amount of work can often be summed up in a single sentence at times, even if it encompasses hundreds of hours

will chew on it but i'm wary of padding it out for the sake of longer paragraphs